src/app/backend/handler/apihandler.go

```diff
@@ -2156,14 +2160,39 @@ func (apiHandler *APIHandler) handleLogs(request *restful.Request, response *res
 		}
 	}
 
-	result, err := container.GetPodLogs(k8sClient, namespace, podID, containerID, logSelector)
+	result, err := container.GetLogDetails(k8sClient, namespace, podID, containerID, logSelector)
 	if err != nil {
 		handleInternalError(response, err)
 		return
 	}
 	response.WriteHeaderAndEntity(http.StatusOK, result)
 }
 
+func (apiHandler *APIHandler) handleLogFile(request *restful.Request, response *restful.Response) {
+	k8sClient, err := apiHandler.cManager.Client(request)
+	if err != nil {
+		handleInternalError(response, err)
+		return
+	}
+	namespace := request.PathParameter("namespace")
+	podID := request.PathParameter("pod")
+	containerID := request.PathParameter("container")
+
+	filename := fmt.Sprintf("attachment; filename='logs-from-%v-in-%v.txt'", containerID, podID)
```

Comment:
I'm not sure about this code related to download. How about adding some interface that will handle this outside of apihandler or using some library like [grab](https://github.com/cavaliercoder/grab)?